### PR TITLE
refactor SnapshotStorage helpers to prepare for later changes

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Problem
Storages are likely to be changed soon.
#### Summary of Changes
Make some helper test methods for common test values to remove noise from more interesting PRs.
Fixes #

### CODE REVIEW

File: runtime/src/accounts_db.rs [hunk 1]
```diff
@@ -5781,7 +5793,7 @@ pub mod tests {
             },
         );
         assert_eq!(result.len(), 2); // 2 chunks
-        assert_eq!(result[0].len(), 0); // nothing found in first slots
+        assert_eq!(result[0].len(), 1); // nothing found in first slots TODO
```
Comment (brooksprumo) [inline runtime/src/accounts_db.rs:5796]: Is there anything left todo here? Or, why did this assert value change?

File: runtime/src/accounts_db.rs [hunk 2]
```diff
@@ -5877,7 +5892,7 @@ pub mod tests {
             },
         );
         assert_eq!(result.len(), 2); // 2 chunks
-        assert_eq!(result[0].len(), 0); // nothing found in first slots
+        assert_eq!(result[0].len(), 256); // nothing found in first slots TODO
```
Comment (brooksprumo) [inline runtime/src/accounts_db.rs:5895]: Same question here about the TODO and changed value.

**Review by brooksprumo — COMMENTED** [review]
The changes look very minimal, but I wasn't sure why the tests values changed though.

File: runtime/src/accounts_db.rs (hunk 1 above)
Comment (jeffwashington) [reply runtime/src/accounts_db.rs:5796]: ah, because I left in the splice changes above. They will be gone soon in a new push.

**Review by brooksprumo — APPROVED** [review]

**Pull request merged**